### PR TITLE
adds volunteer access to edit event hours list for themselves only

### PR DESCRIPTION
**Pull request by bluelightspirit**
also froces admin to edithours page if there is no username inputted, and focrces volunteer to not have ?user= in the link at all for any potential future security isuses.

### CODE REVIEW

**Comment by bluelightspirit** [comment]
story? more like 2 functions though-
as a volunteer i want to change my hours... so i want to be able to view the edit hours page with all the events i attended and access forms to edit hours within those events. these are found in eventList.php and eventHours.php

acceptance criteria:
1. clicking "See & Change My Event Hours" from the home page is meant to show the events attended by themselves only and noone else.
2. clicking "Edit Event" will redirect you to edit hours for that particular event, and you are able to type in field and press change volunteer hours like a normal form

File: index.php [hunk 1]
```diff
@@ -148,6 +148,10 @@
                     <img src="images/new-event.svg">
                     <span>Sign-Up for Event</span>
                 </div>
+                <div class="dashboard-item" data-link="editHours.php">
+                        <img src="images/add-person.svg">
+                        <span>See & Change My Event Hours</span>
```
Comment (jaywynes) [inline index.php]: Maybe the wording here should be "View & Change My Hours", or we can call it the "Hour Manager"

Comment (bluelightspirit) [reply index.php]: hour manager sounds confusing. view & change my hours is ok - though i may want to continue to say event hours to be more specific

File: editHours.php [hunk 2]
```diff
@@ -7,10 +7,13 @@
 $loggedIn = false;
 $accessLevel = 0;
 $userID = null;
+$username = null;
+
 if (isset($_SESSION['_id'])) {
     $loggedIn = true;
     $accessLevel = $_SESSION['access_level'];
     $userID = $_SESSION['_id'];
+    $username = $_SESSION['_id']; // Username is stored here
```
Comment (jaywynes) [inline editHours.php:14]: userID and username are separate variables but you pull the same data, could you wait to assign username until you need it and replace the instances with userID

Comment (bluelightspirit) [reply editHours.php:14]: alright, purging userID as it makes no sense to have

Comment (jaywynes) [reply editHours.php:14]: sounds good!

**Pull request merged**